core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/utilities/DummyTextAnnotationGenerator.java

```diff
@@ -173,135 +190,168 @@
 
         verbSRLArgs_noisy4.put(new IntPair(24, 25), "AM-MOD");
         verbSRLArgs_noisy4.put(new IntPair(26, 28), "AM-MNR");
-
     }
 
-    /**
-     * Generate a basic {@link TextAnnotation} (containing just SENTENCE and TOKENS {@link View}s)
-     * with the same dummy text repeated over a number of sentences.
-     *
-     * @param numSentences The number of sentences that the {@link TextAnnotation} will contain
-     * @return A dummy basic {@link TextAnnotation}
-     */
-    public static TextAnnotation generateBasicTextAnnotation(int numSentences) {
-        int i = 0;
-        List<String[]> docs = new ArrayList<>();
-        while (i < numSentences) {
-            docs.add(basicString.split(" "));
-            i++;
-        }
-        return BasicTextAnnotationBuilder.createTextAnnotationFromTokens(docs);
-    }
 
     private static String[] allPossibleViews = new String[] {ViewNames.POS, ViewNames.LEMMA,
             ViewNames.SHALLOW_PARSE, ViewNames.PARSE_GOLD, ViewNames.SRL_VERB, ViewNames.NER_CONLL,
-            ViewNames.PSEUDO_PARSE_STANFORD };
+            ViewNames.PSEUDO_PARSE_STANFORD};
 
-    public static TextAnnotation generateAnnotatedTextAnnotation(boolean withNoisyLabels) {
-        return generateAnnotatedTextAnnotation(allPossibleViews, withNoisyLabels);
+    /**
+     * Generate a {@link TextAnnotation} (containing a set of {@link View}s specified in the input)
+     * with the some dummy text repeated over a number of sentences.
+     *
+     * @param withNoisyLabels whether to nosify the annotation or not
+     * @param sentenceNum The number of sentences that the {@link TextAnnotation} will contain
+     * @return A dummy basic {@link TextAnnotation}
+     */
+    public static TextAnnotation generateAnnotatedTextAnnotation(boolean withNoisyLabels,
+                                                                 int sentenceNum) {
+        return generateAnnotatedTextAnnotation(allPossibleViews, withNoisyLabels, sentenceNum);
     }
 
     public static TextAnnotation generateAnnotatedTextAnnotation(String[] viewsToAdd,
-            boolean withNoisyLabels) {
-        TextAnnotation ta = BasicTextAnnotationBuilder.createTextAnnotationFromTokens(annotatedTokenizedStringArray);
+                                                                 boolean withNoisyLabels, int sentenceNum) {
+        // at least one sentence
+        // and we can do at-most 3 sentences, for now
+        assert (sentenceNum <= 3 && sentenceNum >= 1);
```

Comment:
I was under the impression that in java `assert`s are always respected and I just learned about this `-ea` option. Any suggestion on how to change it?